cli/providers/aws/credentials.go

```diff
@@ -6,10 +6,15 @@ import (
 	"github.com/pchico83/i2kit/cli/schemas/environment"
 )
 
-func getAWSConfig(e *environment.Environment) (*aws.Config, error) {
+//GetConfig returns a config aws object
+func GetConfig(p *environment.Provider) (*aws.Config, error) {
+	region := p.Region
+	if region == "" {
+		region = "us-west-2"
```

Comment:
I think this should be `p.region`, since you don't use `region` below

Reply:
It is passing region below, in case p.Region is not provided